tuned/plugins/plugin_scheduler.py

```diff
@@ -136,8 +136,230 @@ def get_priority_max(self, sched):
+	`/proc/irq/default_smp_affinity`.
+	--
+	+
+	.An explicit CPU list to set the default IRQ smp affinity to CPUs 0 and 2
+	====
+	----
+	[scheduler]
+	isolated_cores=1,3
+	default_irq_smp_affinity=0,2
+	----
+	====
+	To adjust scheduling policy, priority and affinity for a group of
+	processes/threads, use the following syntax.
+	+
+	[subs="+quotes,+macros"]
+	----
+	group.__groupname__=__rule_prio__:__sched__:__prio__:__affinity__:__regex__
+	----
+	+
+	where `__rule_prio__` defines internal *TuneD* priority of the
+	rule. Rules are sorted based on priority. This is needed for
+	inheritence to be able to reorder previously defined rules. Equal
+	`__rule_prio__` rules should be processed in the order there were
+	defined, however, this is Python interpreter dependant. To disable
+	an inherited rule for `__groupname__` use:
+	+
+	[subs="+quotes,+macros"]
+	----
+	group.__groupname__=
+	----
+	+
+	`__sched__` must be one of:
+	*`f`* for FIFO,
+	*`b`* for batch,
+	*`r`* for round robin,
+	*`o`* for other,
+	*`*`* do not change.
+	+
+	`__affinity__` is CPU affinity in hexadecimal. Use `*` for no change.
+	+
+	`__prio__` scheduling priority (see `chrt -m`).
+	+
+	`__regex__` is Python regular expression. It is matched against the output of
+	+
+	[subs="+quotes,+macros"]
+	----
+	ps -eo cmd
+	----
+	+
+	Any given process name may match more than one group.  In such a case,
+	the priority and scheduling policy are taken from the last matching
+	`__regex__`.
+	+
+	.Setting scheduling policy and priorities to kernel threads and watchdog
+	====
+	----
+	[scheduler]
+	group.kthreads=0:*:1:*:\[.*\]$
+	group.watchdog=0:f:99:*:\[watchdog.*\]
+	----
+	====
+	+
+	The scheduler plug-in uses perf event loop to catch newly created
+	processes. By default it listens to `perf.RECORD_COMM` and
+	`perf.RECORD_EXIT` events. By setting [option]`perf_process_fork`
+	option to `true`, `perf.RECORD_FORK` events will be also listened
+	to. In other words, child processes created by the `fork()` system
+	call will be processed. Since child processes inherit CPU affinity
+	from their parents, the scheduler plug-in usually does not need to
+	explicitly process these events. As processing perf events can pose
+	a significant CPU overhead, the [option]`perf_process_fork` option
+	parameter is set to `false` by default and child processes are not
+	processed by the scheduler plug-in.
+	+
+	The CPU overhead of the scheduler plugin can be mitigated by using
+	the scheduler [option]`runtime` option and setting it to `0`. This
+	will completely disable the dynamic scheduler functionality and the
+	perf events will not be listened to and acted upon. The disadvantage
+	ot this approach is the procees/thread tuning will be done only at
+	profile application.
+	+
+	.Disabling the scheduler dynamic functionality
+	====
+	----
+	[scheduler]
+	runtime=0
+	isolated_cores=1,3
+	----
+	====
+	+
+	NOTE: For perf events, mmapped buffer is used. Under heavy load
```

Comment:
mapped buffer

Reply:
it is actually memory mapped buffer; changed, thank you.